pmd-java/etc/grammar/Java.jjt

```diff
@@ -939,7 +939,7 @@ TOKEN :
 |
   < TEXT_BLOCK_LITERAL:
      "\"\"\"" (<HORIZONTAL_WHITESPACE>)* <LINE_TERMINATOR>
-        ( ~["\"", "\\"] | "\"" ~["\""] | "\"\"" ~["\""] | <STRING_ESCAPE> | "\\" <LINE_TERMINATOR> )*
+        ( ~["\\"] | <STRING_ESCAPE> | <LINE_TERMINATOR> )*
```

Comment:
this is actually too lenient and allows for unescaped `"""` ocurrances in the text block such as:

```java
String s = """
  asd"""asd
  """;
```